orangecontrib/timeseries/widgets/tests/test_owdifference.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+from unittest.mock import Mock
+
+import numpy as np
+
+from Orange.data import Table
+from Orange.widgets.tests.base import WidgetTest
+from orangecontrib.timeseries.widgets.owdifference import OWDifference
+
+class TestOWDifference(WidgetTest):
+    
+    def setUp(self):
+        self.widget = self.create_widget(OWDifference)
+        self.widget.autocommit = True
+        self.data = Table('iris')
+
+    def test_difference(self):
+        w = self.widget
+        w.calc_difference = True
+        w.diff_order = 1
+        w.shift_period = 1
+        self.send_signal(w.Inputs.time_series, self.data[:6])
+        w.selected = ['petal width']
+        w.commit()
+        out = self.get_output(w.Outputs.time_series).X[:, -1]
+        true_out = np.asarray([np.nan, 0, 0, 0, 0, -0.2])
+        np.testing.assert_array_equal(out, true_out)
+
+        # test order always one, if shift > 1
+        w.shift_period = 2
+        w.diff_order = 5
+        self.send_signal(w.Inputs.time_series, self.data[:6])
+        w.selected = ['petal width']
+        w.commit()
+        out = self.get_output(w.Outputs.time_series).X[:, -1]
+        true_out = np.asarray([np.nan, np.nan, 0, 0, 0, -0.2])
+        np.testing.assert_array_equal(out, true_out)
+
+    def test_qoutient(self):
```

Comment:
quotient 😉

Reply:
Thanks :)

Reply:
This doesn't seem to be handled, even though it is marked as resolved?